system_tests/state_fuzz_test.go

```diff
@@ -127,7 +128,7 @@ func (c noopChainContext) GetHeader(common.Hash, uint64) *types.Header {
 }
 
 func FuzzStateTransition(f *testing.F) {
-	f.Fuzz(func(t *testing.T, compressSeqMsg bool, seqMsg []byte, delayedMsg []byte) {
+	f.Fuzz(func(t *testing.T, compressSeqMsg bool, seqMsg []byte, delayedMsg []byte, runMode uint8) {
```

Comment:
I think this will fuzz impossible runmodes, e.g. runmode 10. I think we should limit it to runmodes that actually exist.

Reply:
is there some nice way to fuzz specific "enum" values, or should I use `runMode % numberOfModes`?

Reply:
Not that I know of, modulo makes sense to me